backend/src/utils/graph_db_utils.py

```diff
@@ -43,3 +44,30 @@ def execute_query(llm_query):
         if session:
             session.close()
         driver.close()
+
+
+def populate_db(query, data) -> str:
+    data = {"all_data": data}
+    try:
+        with driver.session() as session:
+            session.run("MATCH (n) DETACH DELETE n")
```

Comment:
Can the clearing be done after 'yield' in app.py? Just wondering if there are benefits to do it one way or another

Reply:
It could be done at the end, yes. Maybe it would be faster for the bootup but I believe it works well at this point for now. I didn't think of doing it that way round, good idea!